ou_dedetai/network.py

```diff
@@ -305,8 +305,7 @@ def logos_reuse_download(
     status_messages: bool = True
 ):
     dirs = [
-        app.conf.install_dir,
```

Comment:
Why was install_dir removed? It's often there

Reply:
As I mentioned in the dev chat, the only file that we download that's ever stored in `install_dir` is the `oudedetai` binary. But it's a relatively small file, and if we're trying to download that, then we are surely doing an update and we don't want to fall back on the existing (and therefore older) version of the binary. So I don't see any reason to include this folder in the existing downloads search.